### PR TITLE
Add link to Google Maps for location

### PR DESCRIPTION
**Pull request by georgeglessner**
This is my first time working with React, so my conventions may be totally off. This was my quick fix for #9. This fix simply adds a link below the "Check Them Out" link that opens up a new tab with the Google Map for the location.

### CODE REVIEW

File: src/components/Brewery.js [hunk 1]
```diff
@@ -16,6 +16,25 @@ export default class Brewery extends Component {
...
   }
 
+  displayLocation(street, city, state) {
+    // format for Google Maps
+    var street = street.replace(/ /g,"+")
+    var city = city.replace(/ /g,"+")
+    var state = state.replace(/ /g,"+")
+    var url = "https://www.google.com/maps/place/"+street+","+city+","+state
+    return (
+      <a
```
Comment (cdrani) [inline src/components/Brewery.js]: If you run `npm run lint` you'll see a slew of errors. These are very easy to correct as you are redeclaring declared variables (street, city, state) and `const` is preferred over `var` as the street, city, and state are constants, meaning they don't get changed after declaration.

To make this code more reusable I would advise moving the following chunk into a separate function:
```js
    var street = street.replace(/ /g,"+")
    var city = city.replace(/ /g,"+")
    var state = state.replace(/ /g,"+")
    var url = "https://www.google.com/maps/place/"+street+","+city+","+state
```
something like this:

```js
  formatAddressUrl(address) {
    const formattedAddress = address
      .map(location => location.replace(/ /g, '+'))
      .join()

    return `https://www.google.com/maps/place/${formattedAddress}/`
  }
```

`address would be an array `[street, city, state]` which be the parameter for the `displayLocation(address)` function.

File: src/components/Brewery.js [hunk 2]
```diff
@@ -42,6 +61,7 @@ export default class Brewery extends Component {
               {city}, {state}
             </h6>
             {this.displayWebsiteUrl(websiteUrl)}
+            {this.displayLocation(street, city, state)}
           </div>
```
Comment (cdrani) [inline src/components/Brewery.js]: The argument for `this.displayLocation(...)` would then be `this.displayLocation([street, city, state])` as mentioned in the comment above.

After these changes and running `npm run lint` you should have one warning and one error from eslint. You can dismiss those as they are or will be addressed in different PR.

Comment (georgeglessner) [reply src/components/Brewery.js]: Thank you for these tips! I have made the mentioned changes. I know this does not totally fix #9, but I thought I would at least assist in getting some sort of map feature up for time being.

**Comment by georgeglessner** [comment]
Thank you for these tips @cdrani ! I have made the mentioned changes. I know this does not totally fix #9, but I thought I would at least assist in getting some sort of map feature up for time being.

**Comment by cdrani** [comment]
Looks Good To Me! Merging in now!
